packages/@contentful--app-scripts/lib/upload/get-upload-settings-args.js

```diff
@@ -0,0 +1,60 @@
+const chalk = require('chalk');
+const ora = require('ora');
+const { getDefinitionById } = require('../definition-api');
+const { getOrganizationById } = require('../organization-api');
+
+const { createClient } = require('contentful-management');
+
+const requiredOptions = {
+  definitionId: '--definition-id',
+  organizationId: '--organization-id',
+  bundleDir: '--bundle-dir',
+  token: '--token',
+};
+
+function validateArguments(options) {
+  Object.entries(requiredOptions).forEach(([option, argument]) => {
+    if (!options[option]) {
+      console.log(
+        `
+  ${chalk.red('Invalid Arguments:')} the argument ${chalk.cyan(argument)} was not defined
+  Run ${chalk.dim('contentful-app-scripts upload --help')} to see all required arguments
```

Comment:
That command only works when the `contentful-app-scripts` are installed globally. If there is a chance that this error is displayed when using the CCA output, we should provide an alternative call here. Even `npx contentful-app-scripts --help` might do it.

Also: This should be `@contentful/app-scripts`

Reply:
True ! I was planning to change that with the real command, changed it now

Reply:
True ! I was planning to change that with the real command, changed it now